src/scripts/main.js

```diff
@@ -1,3 +1,77 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const thead = table.querySelector('thead');
+const tbody = table.querySelector('tbody');
+const headers = thead.querySelector('tr');
+const tr = tbody.querySelectorAll('tr');
+let sorted = [];
+
+function sortStrings(data, key) {
+  const dataTable = [...data];
+
+  const newDataTable = dataTable.sort((a, b) => {
+    const stringA = a.children[key].textContent;
+    const stringB = b.children[key].textContent;
+
+    if (stringA > stringB) {
+      return 1;
+    } else if (stringA === stringB) {
+      return 0;
+    } else {
+      return -1;
+    }
```

Comment:
```suggestion
    if (stringA > stringB) {
      return 1;
    }
    
    if (stringA === stringB) {
      return 0;
    }
    
    return -1;
```
No need for else. You code will not go further after return anyway

Reply:
Not fixed 
<img width="954" alt="image" src="https://user-images.githubusercontent.com/99403198/214125054-75505b7f-974f-4eec-9428-bd23c5bbf060.png">